CMakeLists.txt

```diff
@@ -722,15 +722,6 @@ if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
 
     add_subdirectory(interfaces)
 
-    if(NOT HELICS_BINARY_ONLY_INSTALL AND NOT HELICS_DISABLE_C_SHARED_LIB)
```

Comment:
This part just got moved to the interfaces cmake correct?

Reply:
Yes, it just got moved.